public/app/features/alerting/unified/components/rules/RulesGroup.tsx

```diff
@@ -124,12 +146,12 @@ export const RulesGroup: FC<Props> = React.memo(({ group, namespace, expandAll }
   }
 
   // ungrouped rules are rules that are in the "default" group name
-  const isUngrouped = group.name === 'default';
-  const groupName = isUngrouped ? (
-    <RuleLocation namespace={namespace.name} />
-  ) : (
-    <RuleLocation namespace={namespace.name} group={group.name} />
-  );
+  const groupName =
+    viewMode === 'list' ? (
```

Comment:
Should we exctact `viewMode === 'list'` to a variable?

Reply:
Nit: We could simplify the condition and don't repeat the whole `<RuleLocation>` Component.
```ts
const groupName = viewMode === `list` ? undefined : group.name
```
I think it would be easier not to have to analyse the diff between the two components